expat/tests/runtests.c

```diff
@@ -3778,7 +3778,7 @@ START_TEST(test_ext_entity_ref_parameter) {
   XML_SetParamEntityParsing(g_parser, XML_PARAM_ENTITY_PARSING_ALWAYS);
   XML_SetExternalEntityRefHandler(g_parser, external_entity_ref_param_checker);
   XML_SetExternalEntityRefHandlerArg(g_parser, NULL);
-  handler_data = (void *)g_parser;
+  handler_data = (const void *)g_parser;
```

Comment:
Assignment to `(const void *)` should work from any source type. My vote for dropping the entire cast.

Reply:
Done.